analysers/analyser_merge_defibrillators_FR.py

```diff
@@ -55,7 +55,10 @@ def normalizeHours(self, jours, heures):
     def __init__(self, config, logger = None):
         Analyser_Merge.__init__(self, config, logger)
         self.def_class_missing_official(item = 8370, id = 120, level = 3, tags = ["merge"],
-            title = T_("Defibrillator not integrated"))
+            title = T_("Defibrillator not integrated"),
+            detail = T_("This defibrillator is available through open data but seems to be missing in OpenStreetMap."),
+            fix = T_("Check with ground survey or local knowledge if it is present, and then add it to OSM."),
+            trap = T_("Precision of defibrillator from this database can be very approximative. Be careful when positioning it."))
```

Comment:
Note, all opendata merge already inherit the common doc https://github.com/osm-fr/osmose-backend/blob/master/analysers/Analyser_Merge.py#L857-L865
Only specific here.

Reply:
Hmm. Except trap for GeoDAE database, I'm not sure what can be defined as doc for other defibrillators, I mean it's quite similar to solve compared to other open datasets.

Reply:
Maybe linked to the bad quality of some sources / geocoding, and the need of survey ?

Reply:
OK so no doc by default, and just add doc on dataset we know are of bad quality ? In this case it will be straightforward, as only know about quality of GeoDAE (quite poor), Montfort (good), and AEDMAP (quite good).

Reply:
OK

Reply:
Updated docs accordingly.